### PR TITLE
Copy edits to JS3W1

### PR DESCRIPTION
**Pull request by illicitonion**
## What does this change?

Module: JS3
Week(s): 1

### CODE REVIEW

File: content/js3/blocks/one-to-one/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ emoji= '🧩'
...
 +++
 
-We can now render _any_ film data object in the UI. However, to fully solve this problem we must render a list of data. For each film object, we need to render a corresponding film object in the UI. In this case, there is a {{<tooltip title="one-to-one mapping">}}A **one-to-one mapping** associates every element in a set to exactly one element in another set{{</tooltip>}}between the data array and the UI components on the web page. Each item in the array matches a node in the UI. We can represent this diagrammatically by pairing up the data elements with their corresponding UI components:
+We can now render _any_ film data object in the UI. However, to fully solve this problem we must render a list of data. For each film object, we need to render a corresponding film card in the UI. In this case, there is a {{<tooltip title="one-to-one mapping">}}A **one-to-one mapping** associates every element in a set to exactly one element in another set{{</tooltip>}}between the data array and the UI components on the web page. Each item in the array matches a node in the UI. We can represent this diagrammatically by pairing up the data elements with their corresponding UI components:
```
Comment (SallyMcGrath) [inline content/js3/blocks/one-to-one/index.md:11]:
```suggestion
We can now render _any_ film data object in the UI. However, to fully solve this problem we must render a list of data. For each film object, we need to render a corresponding film card in the UI. Between the data array and the UI components on the web page, there is a {{<tooltip title="one-to-one mapping">}}A **one-to-one mapping** associates every element in a set to exactly one element in another set. {{</tooltip>}}. Each item in the array matches a node in the UI. We can represent this diagrammatically by pairing up the data elements with their corresponding UI components:
```

Interstitial text shouldn't really come in the middle of a sentence. It's not accessible. A screenreader will just read it as it comes, linearly. Can we rephrase? It's a bit clunkier.

**Review by SallyMcGrath — APPROVED** [review]
LGTM

I left a comment on the tooltips you might want to look at.

**Comment by SallyMcGrath** [comment]
@illicitonion I approved this but you didn't merge it. Subsequently, I made large revisions to JS3 1 to address other concerns  in https://github.com/CodeYourFuture/curriculum/discussions/391

I think I made most of your changes but can you look again when you get the chance?

**Comment by illicitonion** [comment]
Yeah, I think this all got folded in - thanks!

**Pull request closed without merging**